specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -2317,6 +2368,13 @@
             "schema": {
               "$ref": "#/definitions/SessionHostPatch"
             }
+          },
```

Comment:
should this be in this PR?

Reply:
yeah we want the force flag for another feature.